WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/CardsAdapter.kt

```diff
@@ -25,6 +27,7 @@ class CardsAdapter(
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CardViewHolder<*> {
         return when (viewType) {
             DashboardCardType.ERROR_CARD.ordinal -> ErrorCardViewHolder(parent)
+            DashboardCardType.TODAYS_STATS_CARD.ordinal -> TodaysStatsCardViewHolder(parent, uiHelpers)
```

Comment:
⚠️: App crashes on switching to a different site with logs:

```
Process: org.wordpress.android.prealpha, PID: 27437
    org.apache.commons.lang3.NotImplementedException: Diff not implemented yet
        at org.wordpress.android.ui.mysite.cards.dashboard.CardsAdapter$DashboardCardsDiffUtil.areItemsTheSame(CardsAdapter.kt:70)
        at androidx.recyclerview.widget.DiffUtil.diffPartial(DiffUtil.java:233)
        at androidx.recyclerview.widget.DiffUtil.calculateDiff(DiffUtil.java:145)
        at androidx.recyclerview.widget.DiffUtil.calculateDiff(DiffUtil.java:105)
        at org.wordpress.android.ui.mysite.cards.dashboard.CardsAdapter.update(CardsAdapter.kt:52)
        at org.wordpress.android.ui.mysite.cards.dashboard.CardsViewHolder.bind(CardsViewHolder.kt:27)
        at org.wordpress.android.ui.mysite.MySiteAdapter.onBindViewHolder(MySiteAdapter.kt:75)
```

Reply:
@ashiagr  My bad 😓 , I missed this scenario while testing the changes. Have implemented the diff util case for `today's stats` card. Thanks for finding this bug. 🙌 . Fixed in 7f8a7d3

Reply:
No worries, actually we were using a mocked response and didn't think of switching sites which could have caught the error in the first place. Thankfully, it was a simple fix. 😄